tests/fixture/issue-2548/case2/output/index.js

```diff
@@ -8,10 +8,17 @@ Object.defineProperty(exports, "X", {
         return _z.default;
     }
 });
-var _exportNames = {
-    X: true
-};
 var _z = _interopRequireWildcard(require("./Z"));
+Object.keys(_z).forEach(function(key) {
+    if (key === "default" || key === "__esModule") return;
+    if (key in exports && exports[key] === _z[key]) return;
+    Object.defineProperty(exports, key, {
```

Comment:
Here is where the call to `Object.defineProperty` can throw the error.  Previously, we injected:

`if (Object.prototype.hasOwnProperty.call(_exportNames, key)) return;`

to guard against this.

Reply:
This is something I'm thinking around, there was previously condition to check if there are multiple exports which couldn't be used with changed.